crates/deltalake-core/src/operations/mod.rs

```diff
@@ -107,60 +107,60 @@ impl DeltaOps {
     /// ```
     #[must_use]
     pub fn create(self) -> CreateBuilder {
-        CreateBuilder::default().with_object_store(self.0.object_store())
+        CreateBuilder::default().with_log_store(self.0.log_store.clone())
```

Comment:
Do we need to `clone()` here (and below)? Since we are consuming the parent stuct, it "should" be possible to pass the log_store_without cloning?

Reply:
Correct on all accounts, fixed soon.